src/features/Auth/SignInForm/SignInForm.module.scss

```
@@ -0,0 +1,22 @@
+@use '@/styles/variables.scss' as *;
+
+.form {
+  width: 100%;
+}
+
+.loginBtn {
+  width: 100%;
+  margin-top: 2rem;
+}
```

다른 페이지에서는 버튼에 `margin-top`이 없던데, 따로 넣은 이유가 있나용?
```suggestion
.loginBtn {
  width: 100%;
}
```

---

실제로 디자인 상으로 다른 페이지들과 다르게 버튼과 인풋바 사이에 간격이 더 멀어서 넣어놨는데 실제 간격좀 다시 보려고 피그마 들어갔는데 로그인 페이지가 없어졌어요..집나간 로그인 페이지 찾습니다..